src/app/store/slices/sharedLinks/index.ts

```diff
@@ -97,8 +97,11 @@ const shareItemWithUser = createAsyncThunk<string | void, ShareFileWithUserPaylo
       if (castedError.message === 'unauthenticated') {
         return navigationService.push(AppView.Login);
       }
+      const userInvited = castedError.message.includes('User already has a role');
```

Comment:
is the code returned by backend a specific number or is it a 500? I say this in case instead of using `the` includes with the message we can use the numerical code of the status of the response

Reply:
It's returning a 400 status, but I don't think it's appropriate for this, because we can get this code for other reasons. Maybe it's a good idea to get a 409 from the backend so we can handle this better.